usmqe_tests/conftest.py

```diff
@@ -311,3 +311,45 @@ def fill_memory():
             raise OSError(stderr)
         return request.param
     return measure_operation(fill_memory)
+
+
+@pytest.fixture(params=[70, 95])
+def workload_swap_utilization(request):
+    """
+    Returns:
+        dict: contains information about `start` and `stop` time of stress-ng
+            command and its `result`
+    """
+    def fill_memory():
+        """
+        Use `stress-ng` tool to stress swap memory for 4 minutes to given
+        percentage
+        """
+        run_time = 240
+        SSH = usmqe.usmssh.get_ssh()
+        host = pytest.config.getini("usm_cluster_member")
+
+        # get total and swap memory of machine via free command
+        meminfo_cmd = "free | awk '{if ($1==\"Mem:\" || $1==\"Swap:\") print $2}'"
```

Comment:
I would suggest to use this alternative way to escape the command, so that it's copy paste friendly and more readable:

```suggestion
        meminfo_cmd = """free | awk '{if ($1=="Mem:" || $1=="Swap:") print $2}'"""
```